hw/dv/tools/dvsim/xcelium.hjson

```diff
@@ -5,7 +5,7 @@
   build_cmd:  "{job_prefix} xrun"
   run_cmd:    "{job_prefix} xrun"
 
-  build_opts: ["-elaborate -64bit -access +r -sv",
+  build_opts: ["-elaborate -64bit -sv",
```

Comment:
Hmm, could you split this and the xcelium_waves change into a separate commit and write a commit message explaining why it's correct? I'm just thinking that someone (maybe me!) is going to want to do archaeology on the history of `xcelium.hjson` sometime in the future.

Reply:
The removal of this switch (duplicated on line 23) has no effect on Xcelium's behavior.

Reply:
Oh, I see! I hadn't realised that it was duplicated. Ok, cool.